lib/ruby_indexer/test/index_test.rb

```diff
@@ -92,6 +92,7 @@ class Something
     end
 
     def test_fuzzy_search
+      @index = Index.new
```

Comment:
This is the same approach as @st0012 took in https://github.com/Shopify/ruby-lsp/pull/2148 but we should really better separate the different indexing behaviours.

Reply:
I don't think it's a good idea to separate different indexing behaviours because then we'll be deviating from what these features are actually going to be used. It'll also make tests harder to maintain in the long term.
 
Now that we also have method names in the PR, we should use this opportunity to decide a few constant/method names that can better avoid false-positive in such tests.